api/lib/opentelemetry/distributed_context/distributed_context.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module DistributedContext
+    # An immutable implementation of the DistributedContext that does not contain any entries.
+    class DistributedContext
+      def entries
+        []
```

Comment:
Should this be a constant so it doesn't allocate an array every time it's called?